thirdparty/tesseract/CMakeLists.txt

```diff
@@ -7,7 +7,9 @@ include("koreader_thirdparty_git")
 
 ep_get_source_dir(SOURCE_DIR)
 
-set(PATCH_CMD sh -c "${ISED} \"s|AM_CONFIG_HEADER|AC_CONFIG_HEADERS|g\" configure.ac")
```

Comment:
I believe so, we do not have AM_CONFIG_HEADER in configure.ac any more, tesseract uses AC_CONFIG_HEADERS now.